packages/cozy-pouch-link/src/PouchManager.js

```diff
@@ -165,9 +176,16 @@ class PouchManager {
         getReplicationURL
       ).then(res => {
         this.addSyncedDoctype(doctype)
-
         logger.log('PouchManager: Replication for ' + doctype + ' ended', res)
-
+        //We want to warmupQueries only once by "start". aka we want to warmup
+        //every time we launch the client (app) but not every time a replication
+        //is done (aka every 30s at least)
```

Comment:
I think it could be clearer.

What do you think of :

```
// Queries are warmed up only once per instantiation of the PouchManager. Since
// the PouchManager lives during the complete lifecycle of the app, warm up
// happens only on app start / restart.
```

Please take the habit of putting a space after the double slash 👼 .

Reply:
spaced comment https://github.com/cozy/cozy-libs/pull/1107 

Thanks 👍